src/test/resources/pages/instructorStudentRecordsPageWithPrivateFeedback.html

```diff
@@ -128,7 +128,7 @@ <h4 class="modal-title">
     <div class="row">
       <div class="col-md-12">
         <br>
-        <div class="student_feedback panel panel-default" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
+        <div class="student_feedback panel panel-default load-feedback-session" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
```

Comment:
Yes, I have disabled it. I am doing one more attempt by merging the latest master and the other step you propose.